recipes/ipapy2/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "ipaPy2" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 8a94928064ea6a97f5b8800f2ff81faaa2b1417f5765a22755a9543102b9cc4e
+
+build:
+  noarch: generic
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - molmass ==2021.6.18
+    - pandas
+    - pip
+    - python
+    - scipy ==1.8.1
+    - tqdm ==4.64.0
```

Comment:
Host should only contain pip and python, for python, please specify a minimum version (3.7 reached end of life by now, so I'd choose 3.8 or 3.9 - or choose the python you are using locally.